src/main/kotlin/com/workos/mfa/MfaApi.kt

```diff
@@ -127,14 +128,14 @@ class MfaApi(private val workos: WorkOS) {
      * Builder class for [ChalllengeFactorOptions].
      */
     class ChallengeFactorOptionsBuilder {
-      private var authenticationFactorId: String? = null
+      private var factorId: String? = null
 
       private var smsTemplate: String? = null
 
       /**
        * Sets the auth factor ID.
        */
-      fun authenticationFactorId(value: String) = apply { authenticationFactorId = value }
```

Comment:
Renaming this is a breaking change.

Is there a reason we can't keep the `authenticationFactorId` naming?

Reply:
I was renaming to follow along the naming style in .NET SDK to keep it similar across the board, can revert back to original naming.